resources/assets/uad_lists.json

```
@@ -5924,7 +5924,7 @@
   {
     "id": "com.samsung.app.newtrim",
     "list": "Oem",
-    "description": "Samsung Video trimmer\nLets you quickly trim video files (from the gallery “Edit -> Studio -> Video Trimmer”)\n",
+    "description": "Samsung Video trimmer\nLet's you quickly trim video files (from the gallery “Edit -> Studio -> Video Trimmer”)\n",
```

```suggestion
    "description": "Samsung Video trimmer\nLets you quickly trim video files (from the gallery “Edit -> Studio -> Video Trimmer”)\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.